concent_api/middleman/constants.py

```diff
@@ -0,0 +1,2 @@
+DEFAULT_INTERNAL_PORT = 8888
```

Comment:
I finally had to think about the ports to define the deployment (https://github.com/golemfactory/concent-deployment/issues/187). I settled upon 9055 for the external port and 9054 for the internal one. These ports are unassigned and do not seem to be in wide use. Please use them as defaults.

Reply:
ok

Reply:
done